uaaextras/tests.py

```diff
@@ -117,11 +119,13 @@ def test_send_email_auth(self, smtplib):
         smtplib.SMTP().login.assert_called_with("user", "pass")
 
     @patch("uaaextras.webapp.render_template")
-    def test_csrf_token(self, render_template):
+    @requests_mock.Mocker(kw="m")
+    def test_csrf_token(self, render_template, m):
         """When a post request is made with a missing or invalid csrf token,
         the error/csrf.html template is displayed
         """
 
+        m.post("https://uaa.bosh-lite.com/introspect", text='{"active": true}')
```

Comment:
This pattern should be added to basically every test where the user is expected to be logged in. 

It's really ugly to have to add it to every test, but from what I can tell, we're stuck with this as long as we're using unittest and not pytest